src/io/flutter/view/FlutterView.java

```diff
@@ -414,36 +415,23 @@ protected void handleJxBrowserInstallationInProgress(FlutterApp app, InspectorSe
     if (jxBrowserManager.getStatus().equals(JxBrowserStatus.INSTALLED)) {
       handleJxBrowserInstalled(app, inspectorService, toolWindow);
     } else {
-      startJxBrowserInstallationWaitingThread(app, inspectorService, toolWindow);
-    }
-  }
+      try {
+        final JxBrowserStatus newStatus = jxBrowserManager.waitForInstallation(INSTALLATION_WAIT_LIMIT_SECONDS);
```

Comment:
why has the executeOnPolledThread call gone away?
The CompletableFuture code as written is still blocking but you could switch to the async version if you don't want to block.